src/bramble/client/ProjectLimiter.js

```diff
@@ -0,0 +1,94 @@
+define(function() {
+"use strict";
+    var ProjectLimiter = {};
+
+    var maxProjectState = {
+            fileSize: 0, // max size of any file in bytes. NOT USED now.
+            fileCount: 0, // total number of files you can have in a project
+            projectSize: 0// total number of bytes for all files in this project
+    };
+    var currentProjectState = {
+        numOfFiles: 0 ,
+        projectSize: 0
+    };
+
+    ProjectLimiter.initMaxLimits =  function (projectState){
```

Comment:
Usually the way you do this is to allow passing a so-called config object, and then you read off the values it has, doing some validation.  For example:

```js
ProjectLimiter.init = function(config) {
    _maxProjectState.fileSize = config.fileSize || 0; // set your default, I used 0
    _maxProjectState.fileCount = config.fileCount || 0; // etc
...
}
```